configs/Lightning-AI/pl-lite_torch-develop.yaml

```diff
@@ -0,0 +1,33 @@
+target_repository:
+  HTTPS: https://github.com/Lightning-AI/lightning.git
+  # checkout a particular branch or a tag
+  checkout: master
+  # define installing package extras
+  install_extras: dev
+  # copy some  test from the target repository
+  copy_tests:
+    - tests/tests_lite
+
+contact:
+  slack:
+    # Adrian Wälchli
+    - URC536Z0W
+
+env:
+  # let's limit the installation only to Pytorch-Lightning
+  PACKAGE_NAME: lite
+
+dependencies:
+  - name: torch
+    install_flags:
+      - "-f https://download.pytorch.org/whl/nightly/torch_nightly.html"
+      - "--pre"
+
+testing:
+  # additional pytest arguments
+  pytest_args: --strict
+
+runtimes:
+  - {os: "ubuntu-22.04", python: "3.10"}
+  - {os: "macOS-12", python: "3.10"}
+#  - {os: "windows-2022", python: "3.10"}
```

Comment:
what's the issue here?

Reply:
I don't think there was an issue, just copied the existing config and adjusted/pruned :otter:

Reply:
let's enable it  then :)

Reply:
```suggestion
  - {os: "windows-2022", python: "3.10"}
```